### PR TITLE
Style/auth screens

### PR DESCRIPTION
**Pull request by zaryan98**
Splash, Sign In, Sign Up Screens created
Unused Code Removed
Need to run npm install

### CODE REVIEW

File: src/components/common/ActionButton.js
Comment (ashartafsol) [inline src/components/common/ActionButton.js:1]: For Text use the font size and Text component from reuseable components

File: src/components/common/DropDown.js
Comment (ashartafsol) [inline src/components/common/DropDown.js:1]: Remove all the unused variables and styling

File: src/components/layout/ScreenBoiler/index.js
Comment (ashartafsol) [inline src/components/layout/ScreenBoiler/index.js:1]: use proper naming conventions for styling. Names should be proper defined

Comment (ashartafsol) [reply src/components/layout/ScreenBoiler/index.js:1]: Screen boiler should have a header component seperately not all the code should be done in this file

File: src/components/view/cards/ContainerListCard.js
Comment (ashartafsol) [inline src/components/view/cards/ContainerListCard.js:1]: No Need to import height and width using dimensions use global accessor R to use the height and width where needed
No inline styling
FlexCont should have proper name 
Images should be imported using global accessor

**Review by ashartafsol — COMMENTED** [review]
Passed

**Review by Maaz003 — APPROVED** [review]

**Pull request merged**